src/paths.ts

```diff
@@ -36,10 +36,13 @@ export function convertDebuggerPathToClient(
     let localPath: string
     if (serverSourceRoot && localSourceRoot) {
         // get the part of the path that is relative to the source root
-        const pathRelativeToSourceRoot = (serverIsWindows ? path.win32 : path.posix).relative(
+        let pathRelativeToSourceRoot = (serverIsWindows ? path.win32 : path.posix).relative(
```

Comment:
So if `serverIsWindows`, first we explicitly use `path.win32`, which always outputs backslashes, and then we replace those again with `path.sep`?